src/sb_pointcloud_processing/src/LineExtractorNode.cpp

```diff
@@ -272,4 +272,4 @@ void LineExtractorNode::getClusterXRange(double& xmin,
 
     xmin = min;
     xmax = max;
-}
\ No newline at end of file
+}
```

Comment:
This is out of the scope of this PR since it should only be dealing with launch files